app/main.py

```diff
@@ -1,8 +1,43 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) -> str:
```

Comment:
Place each argument including self on a new line to not overload one line with argument names and corresponding annotations. Apply changes to this method and to the `__init__` on line 9 as well.